public/toDataByLocation.js

```diff
@@ -3,14 +3,13 @@ export default function toDataByLocation(data) {
   const approvedIndex = headers.findIndex( e => e === 'approved' );
   const stateIndex = headers.findIndex( e => e === 'state' );
   const cityIndex = headers.findIndex( e => e === 'city' );
-  const instructionsIndex = headers.findIndex( e => e === 'instructions' );
```

Comment:
Why is this removed?

Reply:
Cleanup from previous checkin. Linkify code used to be done here; now this value is unused.